R/oauth.R

```diff
@@ -46,7 +46,7 @@ auth_oauth_sign <- function(req, reauth = FALSE) {
       if (is.null(token$refresh_token)) {
         token <- exec(flow, !!!flow_params)
       } else {
-        token <- token_refresh(flow$params$client, token)
+        token <- token_refresh(flow_params$client, token$refresh_token)
```

Comment:
The switch from `flow$params$client` to `flow_params$client` is the immediate fix to what's reported in #166. Once you fix that, a second problem is revealed now that the refresh process can progress farther along:

```
# Error in `query_build()` at httr2/R/req-body.R:219:4:
# ! Query parameters must be length 1 atomic vectors.
# • Problems: refresh_token
```

And that explains the switch from `token` to `token$refresh_token`.